src/style.css

```
@@ -1 +1,82 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: stretch;
+  margin: auto;
+  padding: 0 50px;
+  height: 100%;
+}
```

These styles are useless here, try to delete them and check
```suggestion
.header {
  display: flex;
  justify-content: space-between;
  padding: 0 50px;
}
```